src/config/utils.ts

```diff
@@ -87,3 +89,27 @@ export function replaceAuthErrorsWithEnvironmentVariables(
   message = message.replace('.accessRole', '_ACCESS_ROLE');
   return message;
 }
+
+export function needToLog(
+  currentLoglevel: Loglevel,
+  requestedLoglevel,
```

Comment:
Is there a reason this is not 
```suggestion
  requestedLoglevel: LogLevel,
```
?

Reply:
Fixed